apps/studio/src/lib/codemirror.ts

```diff
@@ -1,13 +1,52 @@
 import CodeMirror from "codemirror";
 
-CodeMirror.defineMIME("text/x-pgsql", {
+export function autoquoteHandler(
+  instance: CodeMirror.Editor,
+  changeObj: CodeMirror.EditorChangeCancellable
+) {
+  if (!instance.getOption("mode").includes("pgsql")) return;
```

Comment:
Quoting aliases isn't a problem in the other languages?

Reply:
@azmy60 quick check-in for this question. If this isn't a problem, feel free to merge plz